.env

```diff
@@ -0,0 +1 @@
+FIGMA_PERSONAL_ACCESS_TOKEN=figd_3ut2fHgX9a_ndVZvLxQM60rlSfytGuz7Q-ZVzNYo
```

Comment:
Should the values in the .env file be shared in the repo?

Reply:
nope!